pkg/porter/list.go

```diff
@@ -40,6 +40,8 @@ func (l CondensedClaimList) Less(i, j int) bool {
 
 // ListBundles lists installed bundles using the printer.Format provided
 func (p *Porter) ListBundles(opts printer.PrintOptions) error {
+	// TODO: supply cnab.Provider interface as second arg, use to access ClaimStore
```

Comment:
Once you move `FetchClaim` like I suggested above, then you will immediate have access to it, because CNABProvider is already on the Porter receiver here.